soundata/datasets/mavd.py

```diff
@@ -0,0 +1,303 @@
+    completeness, or absence of errors. Subject to any liability that may not be excluded or limited by law, 
+    Universidad de la República is not liable for, and expressly excludes, all liability for loss or damage however and 
+    whenever caused to anyone by any use of the MAVD-traffic dataset or any part of it.
+
+
+    Feedback
+    --------
+
+    Please help us improve MAVD-traffic by sending your feedback to: rocamora@fing.edu.uy
+    In case of a problem report please include as many details as possible.
+
+"""
+
+import os
+from typing import BinaryIO, Optional, TextIO, Tuple
+
+import librosa
+import numpy as np
+import csv
+import jams
+import glob
+
+from soundata import download_utils
+from soundata import jams_utils
+from soundata import core
+from soundata import annotations
+from soundata import io
+
+
+BIBTEX = """
+@inproceedings{ZinemanasDCASE2019,
+	Address = {New York, NY, USA},
+	Author = {Zinemanas, P. and Cancela, P. and Rocamora, M.},
+	Booktitle = {Detection and Classification of Acoustic Scenes and Events (DCASE)},
+	Month = {Oct.},
+	Pages = {263--267},
+        Title = {MAVD: a dataset for sound event detection in urban environments},
+	Year = {2019}}
+"""
+
+REMOTES = {
+    "annotations_test": download_utils.RemoteFileMetadata(
+        filename="annotations_test.zip",
+        url="https://zenodo.org/record/3338727/files/annotations_test.zip?download=1",
+        checksum="94e609cad8702f5fb6afe52815547b67"
+    ),
+    "annotations_train": download_utils.RemoteFileMetadata(
+        filename="annotations_train.zip",
+        url="https://zenodo.org/record/3338727/files/annotations_train.zip?download=1",
+        checksum="ea67a330ab695873a12a07ed494eae42"
+    ),
+    "annotations_validate": download_utils.RemoteFileMetadata(
+        filename="annotations_validate.zip",
+        url="https://zenodo.org/record/3338727/files/annotations_validate.zip?download=1",
+        checksum="30edae5c398dffe52a2ee1373e89fd96"
+    ),  
+    "audio_test": download_utils.RemoteFileMetadata(
+        filename="audio_test.zip",
+        url="https://zenodo.org/record/3338727/files/audio_test.zip?download=1",
+        checksum="824b035fa76dc83919e312105db4214f"
+    ),
+    "audio_train": download_utils.RemoteFileMetadata(
+        filename="audio_train.zip",
+        url="https://zenodo.org/record/3338727/files/audio_train.zip?download=1",
+        checksum="c7a897623364f80c8ff3775d8dd601a2"
+    ),
+    "audio_validate": download_utils.RemoteFileMetadata(
+        filename="audio_validate.zip",
+        url="https://zenodo.org/record/3338727/files/audio_validate.zip?download=1",
+        checksum="d220cef03f99b2d108d708421240f86d"
+    )   
+}
+
+LICENSE_INFO = "Creative Commons Attribution 4.0 International"
+
+
+class Clip(core.Clip):
+    """MAVD-traffic Clip class
+
+    Args:
+        clip_id (str): id of the clip
+
+    Attributes:
```

Comment:
Question: do you have metadata about the location or time? If so, that would be interesting to add ass well